threat_intel/util/http.py

```diff
@@ -171,7 +171,7 @@ class MultiRequest(object):
 
     def __init__(
         self, default_headers=None, max_requests=10, rate_limit=0,
-        req_timeout=None, max_retry=10, total_retry=100, drop_404s=False,
+        req_timeout=None, max_retry=10, total_retry=100, drop_404s=True,
```

Comment:
We should set this in the [Virustotal constructor](https://github.com/Yelp/threat_intel/blob/master/threat_intel/virustotal.py#L25) rather then making it a default for all clients (just in case some other APIs threat 404s differently).

Reply:
:++: